### ISSUE TITLE
[STORY] Failure of TBP DICOM node (presumed after patch restart)

### ISSUE THREAD

**Issue by tclose** [Australian-Imaging-Service/pipelines#279]
### Metadata

Epic: #271 
Feature: #273 
Feature Release:
Required knowledge: shallow or mid-level or deep

### Description

The DICOM port of the TBP CTP is going down, presumably after the staging server is restarted after it is patched. Need to ensure that the network configuration is configured after the hypervisor to ensure that it can come back up ok.

### Acceptance Criteria

- [ ] 1. DICOM node is accessible after restart of staging server

### Blocked by

- [ ]

**Comment by tclose** [comment]
duplicate of #272
